### DISCUSSION TITLE
Ancora indicazioni su casi d'uso Matr_005,Matr_013

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#860] [Supporto Webapp] [answered]
Stante l'ultima nota del 5 aprile presente su https://github.com/italia/ansc/discussions/729, parrebbe possibile che il matrimonio con rito civile possa essere celebrato in un luogo di cui il comune **non** abbia disponibilità ed in relazione al quale non esiste alcuna delibera di Giunta che ne dichiari la fruibilità come "casa comunale".
Al momento non ci risulta che la cosa sia possibile né ci è stato possibile individuare alcun riferimento normativo che rappresenti siffatta possibilità se non nel caso in cui sussista:
- impedimento per sposo/a
- imminente pericolo di vita
Ma per queste ultime due circostanze esistono e sono già gestiti casi d'uso specifici (rispettivamente, Matr_002 e Matr_003).

Chiediamo pertanto indicazione chiara e puntuale di quale sia il riferimento normativo in ragione del quale sia possibile, pur in assenza di qualsiasi impedimento documentato, celebrare il matrimonio in un luogo che non sia - per delibera di Giunta - formalmente riconosciuto come "casa comunale".

Rimaniamo in attesa di vostro riscontro e naturalmente a disposizione.

Saluti

**Comment by mttfranci** [comment, marked as answer]
Salve @antonioluberto , come indicato nella issue #729 ANUSCA ci ha conferamto che sono casistiche molto rare ma è stato condiviso di lasciarli. 

Saluti 

Assistenza ANSC
